examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<blockquote>You can also pass <code>microbatch</code> parameter to the <code>train_model</code> action of pipeline to dynamically change splitting strategy. That works even if the model config does not have <code>microbatch</code> key</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>